include/bio/seq_io/reader_options.hpp

```diff
@@ -54,7 +63,7 @@ namespace bio::seq_io
 template <bio::ownership   ownership   = bio::ownership::shallow,
           seqan3::alphabet seq_alph_t  = seqan3::dna5,
           seqan3::alphabet qual_alph_t = seqan3::phred63>
-inline constexpr auto field_types = []()
+inline auto field_types = []()
```

Comment:
We should change this everywhere. It mean we don't need the annoying `std::remove_cvref_t` everywhere when handling this.

Reply:
but if the user specifies a custom field_types object it might be const and we need to strip it anyway no?

Reply:
Why would it be const? I think this is a very specific in-house kind of interface that only very few people target. And if they do, the will use one of the predefined types and if not, they will specify a `bio::ttag<>` as shown in the example. It is really complicated to pass a const-object this way. It will only happen if they explicitly add const which would just be a mistake.